### PR TITLE
Add changelog for 4.4.0

### PR DESCRIPTION
**Pull request by brunnre8**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,192 @@ All notable changes to this project will be documented in this file.
 
 <!-- New entries go after this line -->
 
+## v4.4.0 - 2023-04-22
+
+For more details, [see the full changelog](https://github.com/thelounge/thelounge/compare/v4.3.1...v4.4.0) and [milestone](https://github.com/thelounge/thelounge/milestone/42?closed=1).
+
+This is mostly a developer focused release. Max, Eric and others rewrote the whole thing in typescript / vue3,
```
Comment (MaxLeiter) [inline CHANGELOG.md:11]:
```suggestion
This is mostly a developer focused release. Max, Eric and others rewrote the whole thing in TypeScript / Vue 3,
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,192 @@ All notable changes to this project will be documented in this file.
...
+which should make it much easier to add features and find bugs in the future. So huge kudos from the rest of the team!
+
+Additionally, there's the obvious grab bag of fixes, dependency updates and improvements.  
+Settings are now grouped and hopefully easier to navigate for new users.
+
+Following the nodejs maintenance schedule, The Lounge now needs at least node 14 to run.
```
Comment (MaxLeiter) [inline CHANGELOG.md:17]:
```suggestion
Following the Node.js maintenance schedule, The Lounge now needs at least Node.js 14 to run.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,192 @@ All notable changes to this project will be documented in this file.
...
+
+A huge thanks to everyone who contributed in any way to this release, your help is much appreciated.
```
Comment (MaxLeiter) [inline CHANGELOG.md:19]:
```suggestion
A big thanks to everyone who contributed in any way to this release, your help is much appreciated.
```

you use `huge` earlier

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,192 @@ All notable changes to this project will be documented in this file.
...
 
+## v4.4.0 - 2023-04-22
+
+For more details, [see the full changelog](https://github.com/thelounge/thelounge/compare/v4.3.1...v4.4.0) and [milestone](https://github.com/thelounge/thelounge/milestone/42?closed=1).
+
+This is mostly a developer focused release. Max, Eric and others rewrote the whole thing in typescript / vue3,
+which should make it much easier to add features and find bugs in the future. So huge kudos from the rest of the team!
+
+Additionally, there's the obvious grab bag of fixes, dependency updates and improvements.  
+Settings are now grouped and hopefully easier to navigate for new users.
```
Comment (MaxLeiter) [inline CHANGELOG.md:15]:
```suggestion
Settings are now grouped and easier to navigate for new users.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,192 @@ All notable changes to this project will be documented in this file.
...
+
+Following the nodejs maintenance schedule, The Lounge now needs at least node 14 to run.
+
+A huge thanks to everyone who contributed in any way to this release, your help is much appreciated.
+
+Considering that a bunch of our dependencies had security issues assigned to them, all users are advised to update to the new version.
+
+Packagers: Considering the switch to Typescript, the server build now emits to the dist/ folder.
```
Comment (MaxLeiter) [inline CHANGELOG.md:23]:
```suggestion
Packagers: Considering the switch to TypeScript, the server build now emits to the dist/ folder.  
```

**Review by MaxLeiter — CHANGES_REQUESTED** [review]
left a bunch of nitpicks, the rest LGTM.

**Comment by brunnre8** [comment]
note to self: add xpaw's provenance + lenore's bug fix

**Comment by brunnre8** [comment]
we're now up to date with master, might want to wait for #4728 then do an RC

**Pull request merged**